src/modules/Core/Account/callbacks.js

```diff
@@ -19,42 +19,33 @@ const makeAccountCallbacks = (dispatch: Dispatch): AbcAccountCallbacks => {
     onRemotePasswordChanged: () => console.log('onRemotePasswordChanged'),
 
     onKeyListChanged: () => {
-      console.log('onKeyListChanged')
       // $FlowFixMe
       dispatch(updateWalletsRequest())
     },
 
     onAddressesChecked (walletId: string, progressRatio: number) {
-      if (progressRatio === 1) console.log('onAddressesChecked: ' + walletId + ' ratio:' + progressRatio)
+      if (progressRatio === 1) {
+        console.log(`${walletId} - onAddressesChecked with ratio: ${progressRatio}`)
+      }
     },
 
     onBalanceChanged (walletId: string, currencyCode: string, balance: string) {
-      console.log('onBalanceChanged: ' + walletId + ' ' + currencyCode + ' ' + balance)
+      console.log(`${walletId} - onBalanceChanged for currency${currencyCode}: ${balance}`)
       dispatch(refreshWallet(walletId))
     },
 
     onTransactionsChanged (walletId: string, transactions: Array<AbcTransaction>) {
-      if (transactions && transactions.length) {
-        console.log('onTransactionsChanged length=' + transactions.length.toString())
-        for (const tx of transactions) {
-          console.log('walletId:' + walletId + ' txid:' + tx.txid)
-        }
-      } else {
-        console.log('onTransactionsChanged: No transactions')
+      if (!transactions || !transactions.length) {
```

Comment:
Looks like it would be useful to also have logging when there actually are transactions that have changed

Reply:
I added it back in the next commit